tuning/libtuner.py

```diff
@@ -42,6 +42,9 @@
 device_id = None
 """Do not need to change"""
 
+# Declare special symbols for libtuner to search and locate
+DEVICE_ID_RE = "DEVICE_ID_RE"
```

Comment:
Maybe something more distinct form the surrounding text like `{{DEVICE_ID}}`? Also, I'd call it something like `DEVICE_ID_PLACEHOLDER = "..."`, it doesn't have to be a regular expression